library/core/src/ptr/mod.rs

```diff
@@ -1916,6 +1916,7 @@ pub unsafe fn write_volatile<T>(dst: *mut T, src: T) {
 /// than trying to adapt this to accommodate that change.
 ///
 /// Any questions go to @nagisa.
+#[cfg_attr(not(bootstrap), allow(ptr_to_integer_transmute_in_consts))]
```

Comment:
Given the description of the lint, isn't the comment at https://github.com/rust-lang/rust/pull/130540/files#diff-c175d4e27676febf62c061d31cf9756d256b46e2e44cc6b3177d4ff75e932567R1982-R1986 which causes this to be allowed here incorrect?